lib/core/src/receive_swap.rs

```diff
@@ -2,18 +2,26 @@ use std::{str::FromStr, sync::Arc};
 
 use anyhow::{anyhow, Result};
 use boltz_client::swaps::boltz::RevSwapStates;
+use boltz_client::swaps::boltzv2;
 use log::{debug, error, info, warn};
 use tokio::sync::broadcast;
 
-use crate::ensure_sdk;
 use crate::model::PaymentState::{Complete, Created, Failed, Pending, TimedOut};
-use crate::model::{PaymentTxData, PaymentType, ReceiveSwap};
+use crate::model::{Config, Network, PaymentTxData, PaymentType, ReceiveSwap};
+use crate::{ensure_sdk, utils};
 use crate::{
     error::PaymentError, model::PaymentState, persist::Persister, swapper::Swapper,
     wallet::OnchainWallet,
 };
 
+/// The minimum acceptable fee rate when claiming using zero-conf
+pub const LIQUID_SDK_ZERO_CONF_FEE_RATE_TESTNET: f32 = 0.1;
+pub const LIQUID_SDK_ZERO_CONF_FEE_RATE_MAINNET: f32 = 0.01;
+/// The maximum acceptable amount in satoshi when claiming using zero-conf
+pub const LIQUID_SDK_ZERO_CONF_LIMIT_SAT: u64 = 100_000;
```

Comment:
Perhaps these parameters can be as part of the config?
`zero_conf_min_feerate`
`zero_conf_max_amount`